app/controllers/comments_controller.rb

```diff
@@ -1,21 +1,28 @@
 class CommentsController < ApplicationController
   def create
     @recruit = Recruit.find(params[:recruit_id])
-    current_user.comments.create(comment_params)
-    # flash[:alert] = "Comment can't be empty" unless @comment.save
-    # redirect_back(fallback_location: root_path)
+    comment = @recruit.comments.build(comment_params)
+    comment.user_id = current_user.id
+    respond_to do |format|
+      if comment.save
+        format.js
+      else
+        format.html { render 'recruits/show' }
+      end
```

Comment:
登録に失敗した場合はFlashでメッセージ表示でいいのでは？
これでslackで言ってたエラー出なくなるかと。

```suggestion
      unless comment.save
        flash.now[:alert] = comment.errors.full_messages.join("\n")
      end
      format.js
```

Reply:
修正しましたが直らないので一旦mergeして微調整のブランチで再度修正してみます！